.php_cs.dist

```diff
@@ -10,5 +10,6 @@ return PhpCsFixer\Config::create()
     ->setFinder(
         PhpCsFixer\Finder::create()
             ->in(__DIR__)
+            ->exclude(__DIR__.'/vendor')
```

Comment:
alternatively we could have `->in(['src', 'tests'])` but in any case we want to avoid to fix `vendor` for sure